services/validate.js

```diff
@@ -19,12 +17,19 @@ fs.readdirSync(baseDir)
   })
   .forEach(function (fileName) {
     try {
-      validator.addSchema(fileName, require(path.join(baseDir, fileName)))
+      validator.addSchema(JSON.parse(path.join(baseDir, fileName)), fileName)
```

Comment:
Thanks for the heads-up. I fixed the code. However, the unit tests are still failing. The validation results of `ajv` seem to be fine, but the results of `tv4` do not make sense to me.

Take for example the unit test `PUT /transfers/:id should return 422 if the source amount is zero` in `putTransferSpec.js`. In this unit test, the function [`this.tv4.validateMultiple(data, schema + '.json')`](https://github.com/interledgerjs/five-bells-shared/blob/master/lib/validator.js#L81)  is called with schema=`Transfer` and data:
```
{
    id: '155dff3f-4915-44df-a707-acc4b527bcbd',
    ledger: 'http://localhost',
    debits: [{
        account: 'http://localhost/accounts/alice',
        amount: '0',
        authorized: true
    }],
    credits: [{
        account: 'http://localhost/accounts/bob',
        amount: '10'
    }],
    state: 'executed'
}
```
And the result returned by `tv4` is: `{ errors: [], missing: [], valid: true, schema: 'Transfer' }`, which indicates that `data` complies to `Transfer.json`.

According to [TransferTemplate.json](https://github.com/interledgerjs/five-bells-shared/blob/master/schemas/TransferTemplate.json), the property `id` should be an URI, but `tv4` does not seem to validate this field properly. Even when I remove the property `id` or assign obscure values it gives me the same result as above, e.g. the validation succeeds for `data`:
```
{
    ledger: 'http://localhost',
    debits: [{
        account: 'http://localhost/accounts/alice',
        amount: '0',
        authorized: true
    }],
    credits: [{
        account: 'http://localhost/accounts/bob',
        amount: '10'
    }],
    state: 'executed'
}
```
Surprisingly, if the property `ledger` is not an URI, the validation fails. So `tv4` knows how an URI should look like ...

In contrast, `ajv` produces the expected validation result and, hence, the unit tests are failing.

Am I missing something? Is `tv4` simply not working correctly or is there an explanation for this?